examples/model_selection/plot_roc.py

```diff
@@ -437,7 +439,19 @@
 # global performance of a classifier can still be summarized via a given
 # averaging strategy.
 #
-# Micro-averaged OvR ROC is dominated by the more frequent class, since the
-# counts are pooled. The macro-averaged alternative better reflects the
-# statistics of the less frequent classes, and then is more appropriate when
-# performance on all the classes is deemed equally important.
+# When dealing with imbalanced datasets, choosing the correct metric is crucial.
+# Accuracy may not always reflect model performance well in these cases.
+# For example, metrics such as F1 score, precision, or recall might be more
+# appropriate, depending on the business context or problem you are addressing.
+# It is also essential to select an appropriate averaging method (micro vs. macro)
+# depending on the problem at hand:
```

Comment:
```suggestion
# depending on the desired outcome:
```

Reply:
@lucyleeow, agreed this sounds better, thanks!